languages/en.ini

```diff
@@ -1077,6 +1077,7 @@ Search Tools = "Search Tools"
 Search Type = "Search Type"
 Search within collection = "Search within collection"
 search_AND = "ALL Terms"
+search_backend_partial_failure = "Displaying partial results. Search in at least one of the backends failed."
```

Comment:
Is "backends" perhaps too technical a term to show to the end user? (Not that I necessarily have a better suggestion... :-) ).

Reply:
Yes. I revised the error handling and messages. Not perfect, but now it at least aligns better with empty_search_disallowed wording.

Reply:
Thanks! Would it be best to say target(s) instead of targets in case of singular values?

Reply:
Changed. Not as easy for other languages, but I don't think it's critical. Hopefully the user doesn't see the message too often!

Reply:
Agreed, definitely not a critical problem, just a nice-to-have when the language makes it possible.